.appveyor.yml

```diff
@@ -40,7 +40,7 @@ install:
     - cmd: appveyor-retry conda.exe update --yes --quiet conda
 
 
-    - cmd: appveyor-retry conda.exe install --yes --quiet "conda!=4.6.1,<4.7.11a0" conda-forge-pinning conda-forge-ci-setup=2.* networkx=2.3 "conda-build>=3.16"
+    - cmd: appveyor-retry conda.exe install --yes --quiet "conda!=4.6.1,<4.7.11a0" conda-forge-pinning conda-forge-ci-setup=2.* networkx=2.3 "conda-build>=3.18"
```

Comment:
Changes outside the `recipes/` dir should be made in a separate PR. Would you mind putting this in, @melund? Thanks!

Reply:
@isuruf already fixed this in 6b52315314b1a23048d533fb91cb1857378cc355